modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -164,6 +164,9 @@
 
 #ifdef CONFIG_NRFX_PWM
 #define NRFX_PWM_ENABLED 1
+#ifdef CONFIG_NRF52_ANOMALY_109_WORKAROUND
+#define NRFX_PWM_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
+#endif
 #endif
```

Comment:
This double `#endif` doesn't look nice and there is no need to nest those definitions, so I'd suggest to rather use:
```suggestion
#endif
#ifdef CONFIG_NRF52_ANOMALY_109_WORKAROUND
#define NRFX_PWM_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
#endif
```
or perhaps even better group all those `*_ANOMALY_109_WORKAROUND_ENABLED` symbols in one place:
```
#ifdef CONFIG_NRF52_ANOMALY_109_WORKAROUND
#define NRFX_PWM_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
#define NRFX_SPIS_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
#define NRFX_SPIM_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
#define NRFX_TWIM_NRF52_ANOMALY_109_WORKAROUND_ENABLED 1
#endif
```
This could be updated together with hal_nordic SHA to avoid an extra CI run.

@awojasinski-nordicsemi @nika-nordic What you think?

Reply:
I would agree with @anangl. It would be better if you could group all those symbols together

Reply:
Done